### PR TITLE
Add vodex recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

----

https://github.com/LemonJust/vodex/issues/2

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@LemonJust do you agree with being added as maintainer?

**Comment by LemonJust** [comment]
@goanpeca, I agree with being added as a maintainer! What would I need to do? :)

**Comment by goanpeca** [comment]
>What would I need to do? :)

Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute :)

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/vodex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    # - pip check  # Fails due to pathlib
+  requires:
+    - pip
+
+about:
+  home: https://github.com/LemonJust/vodex
+  summary: | 
+    VoDEx : Volumetric Data and Experiment Manager. Allows to load
+    volumetric data based on experimental conditions.
+  license: MIT
```
Comment (dopplershift) [inline recipes/vodex/meta.yaml:42]:
```suggestion
  license: BSD-3-Clause
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Correct the license metadata and this is good to go.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**